Documentation/ApiOverview/Database/RestrictionBuilder/_ext_localconf.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use MyVendor\MyExtension\Database\Query\Restriction\CustomRestriction;
+
+defined('TYPO3') or die();
+
+if (!isset($GLOBALS['TYPO3_CONF_VARS']['DB']['additionalQueryRestrictions'][CustomRestriction::class])) {
```

Comment:
The whole can be shortened with:

```php
$GLOBALS['TYPO3_CONF_VARS']['DB']['additionalQueryRestrictions'][CustomRestriction::class] ??= [];
```

Reply:
ok, will do

Reply:
Hmm, don't see the change?

Reply:
Sorry, pushing failed. Now the change should be visible